bitcoin/src/blockdata/script/tests.rs

```diff
@@ -33,15 +33,25 @@ fn script() {
 
     // data
     script = script.push_slice(b"NRA4VR"); comp.extend([6u8, 78, 82, 65, 52, 86, 82].iter().cloned()); assert_eq!(script.as_bytes(), &comp[..]);
-    // data push minimality
+    
```

Comment:
This change introduces trailing whitespace. And just because I thought I better check why I care I found this post that sums it up.

https://www.reddit.com/r/vim/comments/1ithiey/why_do_people_care_about_trailing_whitespace/?rdt=42216